scripts/malloc-init.pl

```diff
@@ -1,70 +0,0 @@
-#!/usr/bin/perl
```

Comment:
thanks, I see this file is not called anywhere, so It's ok to remove.
Have you checked th\at there aren't any references for this script anywhere? (articles, usage etc..)